server/api/collections.js

```diff
@@ -454,12 +455,23 @@ router.post("collections.export", auth(), async (ctx) => {
   ctx.assertUuid(id, "id is required");
 
   const user = ctx.state.user;
+  const team = await Team.findByPk(user.teamId);
   const collection = await Collection.scope({
     method: ["withMembership", user.id],
   }).findByPk(id);
   authorize(user, "export", collection);
 
-  const filePath = await archiveCollection(collection);
+  const filePath = await archiveCollections([collection]);
+
+  const stat = await fs.promises.stat(filePath);
+
+  await Export.create({
```

Comment:
This shows up a little weirdly in the table with no url – would you be willing to move this to the same async system?

Reply:
Yeah sure 👍🏻

Reply:
Hey, I reflected on this. Isn't the export table only visible to admins? This is also something that the users have the habit of downloading right away.

Reply:
It's true today but large collections or collections with lots of images lead to the same issues - the file is too big to email and too large to generate sync on the http server.

I think it would be a reasonable compromise to make this functionality only available to admins to make it reliable